eclipse/src/saros/ui/util/CollaborationUtils.java

```diff
@@ -460,4 +464,27 @@ private static void refreshProjects(
 
     progress.done();
   }
+
+  private static List<JID> ensureFQJID(final Collection<JID> jids) {
+
+    final List<JID> result = new ArrayList<>();
+
+    for (final JID jid : jids) {
+      if (jid.isResourceQualifiedJID()) {
+        result.add(jid);
+        continue;
```

Comment:
Use else block instead of this `continue` to make control flow more apparent? Probably a matter of taste.

Reply:
I guess it more useful to display all invalid JID at once in case you are trying to invite 30. Each invalid contact will spawn a dialog, one after another.